### PR TITLE
Add to playlist

### PR DESCRIPTION
**Pull request by abhishektanwar**

### CODE REVIEW

**Comment by abhishektanwar** [comment]
FEATURES ADDED :
- Add video to playlist from video card (dropdown)
- Add video to playlist from individual video play page
- Create a new playlist from playlists listing page
- Remove video from playlist 

Files to check
- src/components/PlaylistCard/index.jsx
- src/components/PlaylistCreation/index.jsx
- src/screens/Playlist/index.jsx
- src/screens/Playlists/index.jsx

File: src/components/PlaylistCard/playlist-card.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  margin-left: 10px;
+}
+.empty-playlist-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 82%;
+}
+
+/* #21202070 */
```
Comment (Ananyamadhu08) [inline src/components/PlaylistCard/playlist-card.css]: you can delete the unnecessary  commented code.

File: src/screens/Playlists/index.jsx [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
+      </div>
+      <div>
+        {playlists.length === 0 ? (
+        <div className="flex-row flex-justify-content-center margin-top-20">
+          <h3>You do not have any playlist yet.</h3> 
+        </div>): (
+
+          <div className="playlist-grid-container">
+          {playlists.map((playlist)=>{
+            // console.log("playlist",playlist)
```
Comment (Ananyamadhu08) [inline src/screens/Playlists/index.jsx]: can delete commented code.

**Review by Ananyamadhu08 — APPROVED** [review]
Your code is well structured and clean. The UI for the video library is clean and minimalistic, one thing you can do is is delete the commented code in you project. Overall good job Abhishek! You can go ahead and merge 🚀

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Clean code. Easy to understand.

Good to merge 👍🏻

File: src/components/PlaylistCard/playlist-card.css (hunk 1 above)
Comment (abhishektanwar) [reply src/components/PlaylistCard/playlist-card.css]: removed.

File: src/screens/Playlists/index.jsx (hunk 2 above)
Comment (abhishektanwar) [reply src/screens/Playlists/index.jsx]: removed.

**Pull request merged**